<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick a color, man!</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QSpinBox" name="red_spinbox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="red_slider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="green_slider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="green_label">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="green_spinbox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="red_label">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="blue_label">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="blue_slider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="blue_spinbox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="colorDisplay" native="true">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>red_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>red_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>red_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>red_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>green_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>green_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>green_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>green_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blue_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blue_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blue_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blue_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
